openmdao/docs/openmdao_book/features/building_blocks/components/spline_comp.ipynb

```diff
@@ -264,8 +264,8 @@
     "otherwise empty dictionary and passing it in as 'interp_options'.\n",
```

Comment:
<p>Minor issue, but spacing between "thestarting", "controlpoints" and "firstand/or" and "the'interp_options'" is missing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/3338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Fixed